### PR TITLE
Kong GW quickstart remove C* config and use PG_HOST

### PR DESCRIPTION
**Pull request by rspurgeon**
### Summary
Changes the Kong quickstart guide to use the `PG_HOST` value to configure the database connection instead of the deprecated C* value.

### Reason
C* is deprecated, PG values should be used

### Testing
Tested the code in a local docker environment

### CODE REVIEW

File: app/gateway/2.8.x/get-started/quickstart/index.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ For a comprehensive list of installation options, see our [Install page](/gatewa
         -e "KONG_PG_HOST=kong-database" \
         -e "KONG_PG_USER=kong" \
         -e "KONG_PG_PASSWORD=kong" \
-        -e "KONG_CASSANDRA_CONTACT_POINTS=kong-database" \
+        -e "KONG_PG_HOST=kong-database" \
```
Comment (mheap) [inline app/gateway/2.8.x/get-started/quickstart/index.md:61]: Duplicate for line 58

**Review by mheap — DISMISSED** [review]
Added comments inline

File: app/gateway/2.8.x/get-started/quickstart/index.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ For a comprehensive list of installation options, see our [Install page](/gatewa
      -e "KONG_PG_HOST=kong-database" \
      -e "KONG_PG_USER=kong" \
      -e "KONG_PG_PASSWORD=kong" \
-     -e "KONG_CASSANDRA_CONTACT_POINTS=kong-database" \
+     -e "KONG_PG_HOST=kong-database" \
```
[the remark this answers is not in the document]
Comment (rspurgeon) [reply app/gateway/2.8.x/get-started/quickstart/index.md]: 🤦 thank you @mheap

File: app/gateway/2.8.x/get-started/quickstart/index.md (hunk 1 above)
Comment (rspurgeon) [reply app/gateway/2.8.x/get-started/quickstart/index.md]: 🤦 thank you @mheap

File: app/gateway/2.8.x/get-started/quickstart/index.md [hunk 3]
```diff
@@ -2,12 +2,12 @@
 title: Start Kong Gateway
 ---
 
-In this section, you'll learn how to install and manage your Kong Gateway instance. First, you'll start Kong Gateway to gain access to its Admin
+In this section, you'll learn how to install and manage your {{site.base_gateway}} instance. First, you'll start {{site.base_gateway}} to gain access to its Admin
 API, where you'll manage entities including Services, Routes, and Consumers.
```
Comment (cloudjumpercat) [inline app/gateway/2.8.x/get-started/quickstart/index.md:6]:
```suggestion
API, where you'll manage entities including services, routes, and consumers.
```

**Review by cloudjumpercat — APPROVED** [review]
LGTM!

**Pull request merged**